packages/app/lib/nativeDb.ts

```diff
@@ -73,6 +74,10 @@ export class NativeDb extends BaseDb {
     this.connection.delete();
     this.connection = null;
     this.client = null;
+
+    // reset values related to tracking db sync state
+    await kv.headsSyncedAt.resetValue();
```

Comment:
we should probably add this to `purgeDb()` on in `electronDb`, too, since we actually use these values there

Reply:
good catch!